atlp-rwanda/dynamites-ecomm-be
# chore(test): set test runner

#### What does this PR do?
- Setting test runner
#### Description of Task to be completed?
- Set jest as test runner
#### How should this be manually tested?
- Clone this repo `git clone https://github.com/atlp-rwanda/dynamites-ecomm-be.git`
- Change your directory `cd dynamites-ecomm-be`
- Install dependencies `yarn`
- put environment variables
- run tests locally `yarn test`

Expected outcomes
> Running tests with 80% as coverage
#### Any background context you want to provide?
- This project to work need `node` version 18
#### What are the relevant issue id?
- #8 
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
